src/app/services/userService.tsx

```diff
@@ -31,3 +31,13 @@ export const updateUserProfile = async ({
   if (!response.ok) throw new Error("Error updating profile");
   return response.json();
 };
+
+export const fetchNotifications = async(userId: string) => {
```

Comment:
Type the return value of the function
```suggestion
export const fetchNotifications = async(userId: string): string[] => {
```